pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
 import time
-from pokemongo_bot import logger
 from pokemongo_bot.human_behaviour import (normalized_reticle_size, sleep,
                                            spin_modifier)
+from pokemongo_bot.cell_workers.base_task import BaseTask
 
-class PokemonCatchWorker(object):
+class PokemonCatchWorker(BaseTask):
```

Comment:
PokemonCatchWorker is *not* a worker. It is essentially an API helper method that other workers use. It needs to be pulled out of this folder, but that shouldn't be done in this PR. This line definitely needs to get changed back though.